modules/heat_conduction/test/tests/NAFEMS/transient/T3/nafems_t3_edge2_fine.i

```diff
@@ -0,0 +1,102 @@
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 1
+  nx = 10
+  xmin = 0.0
+  xmax = 0.1
+  elem_type = EDGE2
+[]
+
+[Variables]
+  [./temp]
+    order = FIRST
```

Comment:
order, family, and initial_conditition are all default values here, probably not necessary?